modules/Template/PortalDataset.yaml

```
@@ -72,6 +72,10 @@ classes:
       series:
         range: string
         required: false
+      externalViewer:
```

```suggestion
      visualizeDataOn:
```

Suggesting from @jaybee84